src/main/java/uk/gov/hmcts/juror/api/moj/domain/User.java

```diff
@@ -76,7 +76,7 @@ public class User implements Serializable {
     @Enumerated(EnumType.STRING)
     private UserType userType;
 
-    @ElementCollection(fetch = FetchType.EAGER)
```

Comment:
was there a reason this was eager?

Reply:
When the User object was created it was assume when ever you grab the user you will also want to grab the roles. As we integrated this more throughout the application it turns out this was not the case so we no longer need it to EAGER load